scripts/k8s/prometheus-local-port-forward.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+
+LOCAL_PORT=${LOCAL_PORT:-9090}
+NAMESPACE="${NAMESPACE:-stackrox}"
+
+CENTRAL_POD=""
+echo -n "Waiting for a running Central pod"
+until [ -n "${CENTRAL_POD}" ]; do
+  echo -n '.'
+  sleep 2
+  CENTRAL_POD="$(kubectl get pod -n $NAMESPACE --selector 'app=central' | grep Running | awk '{print $1}')"
+done
+echo "found Central pod: ${CENTRAL_POD}"
+
+echo -n "Setting up Prometheus port-forwarding..."
+
+# kubectl should be killed whenever this script is killed
+trap 'kill -TERM ${PID}; wait ${PID}' TERM INT
+kubectl port-forward -n "$NAMESPACE" "$CENTRAL_POD" "${LOCAL_PORT}:9090" > /dev/null &
+PID=$!
+
+until $(curl --output /dev/null --silent --fail -k "http://localhost:${LOCAL_PORT}/metrics"); do
```

Comment:
```suggestion
until curl --output /dev/null --silent --fail -k "http://localhost:${LOCAL_PORT}/metrics"; do
```
https://www.shellcheck.net/wiki/SC2091